site/en/docs/webstore/using_webstore_api/index.md

```diff
@@ -2,36 +2,52 @@
 layout: "layouts/doc-post.njk"
 title: "Using the Chrome Web Store Publish API"
 #date: TODO
-#updated: TODO
+updated: 2021-12-21
 description: How to programmatically create, update, and publish items in the Chrome Web Store.
 ---
 
 ## Overview {: #overview }
 
 The Chrome Web Store Publish API provides a set of REST endpoints for programmatically creating,
-updating, and publishing items in the Chrome Web Store. Using this API, you can automate the process
-of uploading and publishing items into the store.
-
-## Before you begin {: #beforeyoubegin }
-
-To use the Chrome Web Store Publish API, you need to enable the API for your project in the [Google
-Developers Console][1].
-
-1.  Visit the [Google Developers Console][1].
-2.  Create a new project or select an existing one.
-3.  In the sidebar on the left, select **APIs & auth**.
-4.  In the displayed list of available APIs, set the status of the Chrome Web Store API to **ON**.
-5.  Accept the Terms of Service.
-6.  In the sidebar on the left, select **Credentials**.
-7.  Find the lines labeled **Client ID** and **Client secret**. Note that there may be a client ID
-    without a client secret for use with Compute Engine and App Engine. In that case, create a new
-    client ID and client secret.
-8.  To create the client ID and client secret, click on **CREATE CREDENTIALS**, select **OAuth client ID** and select **Desktop app** under **Application type**.
-9.  Get an access token:
-
-Once you have the client ID and client secret, you can retrieve an access token to work with the
-API. For example, enter this URL in your browser, replacing the \$CLIENT_ID with the one for your
-app:
+updating, and publishing items in the Chrome Web Store.
+
+## Initial setup {: #setup }
+
+Take the following steps to get the access keys needed:
+
+### Enable the Chrome Web Store API {: #enable-cws-api }
+
+1. Go to the [Google Cloud Console][google-dev-console].
+1. Create a new project or select an existing one.
+   {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/g5o7qpsDv5kKiQmlCwkv.png", alt="Create a new
+   project in the Google Console", width="600", height="149" %}
+1. In the **search bar** type “Chrome Web Store API”.
+1. Enable the **Chrome Web Store API**.
+
+### Configure the OAuth consent screen  {: #oauth-setup }
```

Comment:
@AmySteam, I revised this line to match my suggestion here: https://github.com/GoogleChrome/developer.chrome.com/pull/1860/files/59708cac9a503de00b29ecc9b6ce9d04fa59a550#r774817922 Please resolve this comment if you're comfortable with this change.